api4/channel.go

```diff
@@ -167,6 +167,10 @@ func updateChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 		oldChannel.Type = channel.Type
 	}
 
+	if channel.GroupConstrained != nil {
```

Comment:
@mgdelacroix Could you jog my memory on why this only gets updated from a `nil` value? Does this mean the value, once set, isn't updatable?